ci/labeler.yaml

```diff
@@ -0,0 +1,233 @@
+# This is the Jenkins ci variant of the .github/labler.yaml
+# Instead of folder based glob strings, this one uses regex strings
```

Comment:
should we add a comment in .github/labeler.yaml that it is deprecated. 
And that manual setting of test-spec/test-labels won't be supported moving forward.